core/process.py

```
@@ -146,6 +157,6 @@ def terminate_process_gracefully(
         # Try one last time with kill
         try:
             process.kill()
-        except:
+        except Exception:
             pass
```

Ruff (0.15.20)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(Exception): ...`

(SIM105)